md/templates/md.html

```diff
@@ -15,6 +15,19 @@
   <img src="{% static "svg/noun_19618_cc-white.svg" %}" class="state md">
 {% endblock state-header-graphic %}
 
+{% block agency-intro %}
+  <p class='nc-agency-intro'>
+    Review agency-level data on the racial and ethnic demographics of people
+    stopped, searched, and subjected to force in the course of traffic stops
```

Comment:
maybe this is a Katie question: can we remove the "subjected to force" text for MD?  (we don't have data on that)